### ISSUE TITLE
Long image alt text protrudes into editor when image is selected

### ISSUE THREAD

**Issue by b2m** [ckeditor/ckeditor-dev#898]
## Are you reporting a feature request or a bug?

Bug

## [Check if the issue is already reported](http://docs.ckeditor.com/#!/guide/dev_issues_readme-section-avoid-duplicates)

None found.

## Provide detailed reproduction steps (if any)

There is a bug concerning long texts in the alt attribute of images in combination with the uploadimage plugin.

1. open https://codepen.io/anon/pen/oGNbKQ?editors=1010
2. select image by clicking on it

### Expected result

Image should be selected, nothing more.

### Actual result

The content of the alt attribute is copied to the hidden selection (ok).
But the text it is so long that the hidden selection it is no longer hidden but partly visible to the user (not ok).

## Other details

* Browser: Chrome, Firefox
* OS: independent
* CKEditor version: 4.7.2
* Installed CKEditor plugins: image2, uploadimage

**Comment by mlewand** [comment]
Our trick did not expect that the alt will be as long as it is. 🙂 Generally you wan't to keep alt as short and meaningful as possible, but ofc this is a bug.

For the implementer/reviewer: it's crucial to ensure accessibility after the fix. Ensure that the screen reader reads labels for widgets. There should be a manual test for it somewhere as I recall we were working with not that long ago.
